config/ember-try.js

```diff
@@ -10,18 +10,6 @@ module.exports = function() {
   ]).then((urls) => {
     return {
       scenarios: [
-        {
-          name: 'ember-lts-2.16',
```

Comment:
Can you remove the scenario from the travis config?

Reply:
certainly! thanks for catching it @offirgolan.